DEVELOPMENT.md

```diff
@@ -117,21 +117,37 @@ Your user must be a cluster admin to perform the setup needed for Knative.
 
 The value you use depends on
 [your cluster setup](https://www.knative.dev/docs/install/): when using
-Minikube, the user is your local user; when using GKE, the user is your GCP
-user.
+Minikube or the Kubernetes under local Docker desktop, the user is your local user;
+when using GKE, the user is your GCP user.
 
 ```shell
 # For GCP
 kubectl create clusterrolebinding cluster-admin-binding \
   --clusterrole=cluster-admin \
   --user=$(gcloud config get-value core/account)
 
-# For minikube
+# For minikube or Kubernetes under docker-desktop
 kubectl create clusterrolebinding cluster-admin-binding \
   --clusterrole=cluster-admin \
   --user=$USER
 ```
 
+### Resource allocation for Kubernetes
+
+Please allocate sufficient resources for Kubernetes, especially when you run a Kubernetes cluster on your local machine.
```

Comment:
So I like this paragraph as the resource requirements could be easy to miss especially if you had already setup your cluster (locally or not). However, the "For ..." sections below this feel like a repeat of https://github.com/knative/serving/blob/master/DEVELOPMENT.md#create-a-cluster-and-a-repo. 

I would rather see us link to the section above in the DEVELOPMENT document or link to the base install page https://knative.dev/docs/install/ where all the options are enumerated. 

Also FYI `knative.dev` is the website version of `github.com/knative/docs`. We should prefer to link to the website over the repository for ease of navigation and reading.

Reply:
Thx. Done.